command/certificate/remote.go

```diff
@@ -56,20 +63,27 @@ func getPeerCertificates(addr, serverName, roots string, insecure bool) ([]*x509
 // trimURL returns the host[:port] if the input is a URL, otherwise returns an
 // empty string (and 'isURL:false').
 //
+// If the URL is valid and no port is specified, the default port determined
+// by the URL prefix is used.
+//
 // Examples:
-// trimURL("https://smallstep.com/onbaording") -> "smallstep.com", true, nil
+// trimURL("https://smallstep.com/onbaording") -> "smallstep.com:443", true, nil
 // trimURL("https://ca.smallSTEP.com:8080") -> "ca.smallSTEP.com:8080", true, nil
 // trimURL("./certs/root_ca.crt") -> "", false, nil
-// trimURL("hTtPs://sMaLlStEp.cOm") -> "sMaLlStEp.cOm", true, nil
+// trimURL("hTtPs://sMaLlStEp.cOm") -> "sMaLlStEp.cOm:443", true, nil
 // trimURL("hTtPs://sMaLlStEp.cOm hello") -> "", false, err{"invalid url"}
 func trimURL(ref string) (string, bool, error) {
 	tmp := strings.ToLower(ref)
-	for _, prefix := range urlPrefixes {
+	for prefix, _ := range urlPrefixes {
```

Comment:
```suggestion
	for prefix := range urlPrefixes {
```
This is what the linter is looking for